qtawesome/iconic_font.py

```diff
@@ -29,6 +29,52 @@
 # use system fonts
 SYSTEM_FONTS = False
 
+# Needed imports and constants to install bundled fonts on Windows
+# Based on https://stackoverflow.com/a/41841088/15954282
+if os.name == 'nt':
+    import shutil
+    import ctypes
```

Comment:
Let's move these two imports above (i.e. to the `Standard library imports` section) because they exist in all OSes.